docs/sources/flow/monitoring/debugging.md

```diff
@@ -89,4 +100,34 @@ The location of Grafana Agent's logs is different based on how it is deployed.
 Refer to the [`logging` block][logging] page to see how to find logs for your
 system.
 
+## Debugging clustering issues
+
+To debug issues when using [clustering][], check for the following symptoms.
+
+- **Cluster not converging**: The cluster peers are not converging on the same
+  view of their peers' status. This is most likely due to network connectivity
+issues between the cluster nodes. Use the Flow UI to understand which nodes are
+not being picked up by their peers.
```

Comment:
This is implying you have to look at the Flow UI for every node, right? I think we should note that here.